config.json

```diff
@@ -141,7 +141,11 @@
       "core": false,
       "difficulty": 3,
       "slug": "perfect-numbers",
-      "topics": null,
+      "topics": [
+	"integers",
+	"sequences",
+	"mathematics"
```

Comment:
Great suggestions! :) The only one I'm not sure about is `sequences`. How would you say that relates to this exercise? 

Also, `enumerations` and `exception_handling` would be good topics since the exercise uses an enum and the tests expect an exception to be thrown :)

And a minor thing, these should all be indented two spaces in from `topics`, like the other exercises that have topics (e.g. `hello-world` or `two-fer`) :)

Reply:
Thanks for the advice. 
About `sequences`, I was thinking about how the exercise filters a sequence of integers, but in fact, `sequences` is supposed to refer to a data type as stated in the topic list. So `sequences` shouldn't be included. Also will fix the indentation.
I will include `enumerations` and `exception_handling`. 

I also want to propose `filtering` since multiples of the `naturalNumber` are being filtered from the `IntStream`, according to the reference file.

Reply:
Great :) Sure, you can add `filtering` if you think that would be useful :)